packages/server-wallet/src/models/channel.ts

```diff
@@ -152,6 +163,24 @@ export class Channel extends Model implements RequiredColumns {
     };
   }
 
+  get channelResult(): ChannelResult {
+    const {channelId, participants, appDefinition, supported} = this;
+    if (!supported) throw 'not supported';
+    const {outcome, appData, turnNum} = supported;
+
+    const allocations = serializeAllocation(checkThat(outcome, isAllocation));
+
+    return {
+      channelId,
+      participants,
+      appData,
+      allocations,
+      appDefinition,
+      status: 'funding', // FIXME
```

Comment:
Do we have a specification written somewhere for what defines these status types? I actually forget what `funding` means but I'm guessing it's that this channel is not yet funded, but there exists some objective the wallet is carrying out, whether direct or virtual, to fund it.

Reply:
I'm unaware of one, which is why this is hard coded. There definitely needs to be one.